test/files/avs-ascii.fld

```diff
@@ -0,0 +1,9 @@
+# AVS field file (ascii variant)
```

Comment:
Just a question. Does all AVS field file has this `# AVS`?

Reply:
Every one that I have seen.  This format was never blessed by any standards organization and I have been unable to find online documentation of it.  (I learned the format from a paper manual before the WWW existed!)  So I think we will not find any 100% definitive answer to the question.
I already added a comment about this assumption in the long README here:
https://github.com/JeffFessler/AVSfldIO.jl
That README is probably more complete documentation than you will find anywhere on the web!

It is trivial to add that header if someone (1) has a file without it and (2) wants to read it with Julia.
If at some later point there is someone who has a big pile of the data format without that header then I am happy to write a fancier format detector function if needed.

I have added the query test in the most recent commit.
I also changed the format name from "FLD" to "AVSfld" to be more specific.